src/errors/validation_exception.rs

```diff
@@ -341,6 +341,20 @@ impl ValidationError {
     fn __str__(&self, py: Python) -> String {
         self.__repr__(py)
     }
+
+    fn __reduce__(slf: &PyCell<Self>) -> PyResult<(&PyAny, PyObject)> {
+        let py = slf.py();
+        let callable = slf.getattr("from_exception_data")?;
+        let borrow = slf.try_borrow()?;
+        let args = (
+            borrow.title.as_ref(slf.py()),
```

Comment:
Should this be
```suggestion
            borrow.title.as_ref(py),
```
like the rest?

Reply:
Yes 👍